spec/models/recurring_event_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe RecurringEvent, type: :model do
+  subject { build(:recurring_event) }
+
+  it { is_expected.to be_valid }
+  it { should respond_to :day }
+  it { is_expected.to respond_to :week }
+  it { should belong_to :event }
```

Comment:
NITPICK: avoid mixing the `should` syntax with `expect`. try to use only `expect` here because `should` is deprecated.